src/adapter/src/catalog.rs

```diff
@@ -1547,7 +1547,8 @@ pub struct Database {
     pub schemas_by_id: BTreeMap<SchemaId, Schema>,
     pub schemas_by_name: BTreeMap<String, SchemaId>,
     pub owner_id: RoleId,
-    pub privileges: Vec<MzAclItem>,
+    // Key is the role granted the privilege, value is the privilege itself.
```

Comment:
```suggestion
    // Key is the role that granted the privilege, value is the privilege itself.
```

Reply:
I updated all occurrences of this comment.